### PR TITLE
feat(drag-handle): support in edgeless mode

### PR DESCRIPTION
**Pull request by fundon**
Related to: #1704

* [x] Support drag handle in edgeless mode inside one frame.
* [x] Support drag handle in edgeless mode across multiple frames (or dragging outside of the original frame).
* [x] Press `Esc` will cancel the operation.
* [x] tests

https://user-images.githubusercontent.com/27926/226121102-f48a4a98-f4c4-4239-97ba-c764743255cd.mov

### CODE REVIEW

**Comment by fundon** [comment]
@doodlewind In which mode is `drag-handle` enabled?

**Comment by doodlewind** [comment]
> @doodlewind In which mode is `drag-handle` enabled?

I think it should be the default edgeless mouse mode.

/cc @alt1o

File: packages/store/src/workspace/page.ts [hunk 1]
```diff
@@ -507,6 +507,45 @@ export class Page extends Space<FlatBlockMap> {
     nextParentModel.propsUpdated.emit();
   }
 
+  @debug('CRUD')
+  moveBlocksToParent(blocks: BaseBlockModel[], parentId: string) {
```
Comment (doodlewind) [inline packages/store/src/workspace/page.ts:511]: Can we reuse the existing `moveBlocks` method?

Comment (fundon) [reply packages/store/src/workspace/page.ts:511]: https://github.com/toeverything/blocksuite/blob/75173db962085c38d7e676498dc755b96cfbd3ea/packages/store/src/workspace/page.ts#L483-L486

`targetModel` needs to get its parent model first.

But in this case, we already know the parent model (frameId);

**Review by doodlewind — APPROVED** [review]

File: packages/store/src/workspace/page.ts (hunk 1 above)
Comment (doodlewind) [reply packages/store/src/workspace/page.ts:516]: I think `moveBlocks` should be refactored based on your usage

Comment (fundon) [reply packages/store/src/workspace/page.ts:516]: I try to refactor it. :)

**Pull request merged**